org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/resources/admin/RepositoryAdminResource.java

```diff
@@ -67,7 +67,7 @@ public Response onGet(
 			return Response.noContent().build();
 		} else if (commit != null) {
 			try {
-				((GitBasedRepository) Prefs.INSTANCE.getRepository()).addCommit();
+				((GitBasedRepository) Prefs.INSTANCE.getRepository()).addCommit(null);
```

Comment:
Please also add another `addCommit` method. Maybe passing `commit through admin resource` as commit message. One really should not pass `null`.